liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/jsp/CompileJspMojo.java

```diff
@@ -16,20 +19,20 @@
 /**
  * Compile the JSPs in the src/main/webapp folder.
  * 
- * @goal compile-jsp
- * 
- * @phase compile
- * 
- * @requiresDependencyResolution compile
+ * @Mojo( name = "compile-jsp", defaultPhase = LifecyclePhase.COMPILE, 
+ *        requiresDependencyResolution = ResolutionScope.COMPILE)   
  *
  */
+@Mojo( name = "compile-jsp", defaultPhase = LifecyclePhase.COMPILE, 
+       requiresDependencyResolution = ResolutionScope.COMPILE)  
 public class CompileJspMojo extends BasicSupport {
 
     /**
      * The version of JSP that should be compiled against. Defaults to 2.3. Can be 2.2 or 2.3
      * 
-     * @parameter
+     * @Parameter( property="jspVersion")
      */
+	@Parameter( property="jspVersion")
```

Comment:
This is just one of many incidences. If you have one space after open bracket, you should also have one space before closing bracket.  It happens in multiple place, place fix them.